chainerx_cc/chainerx/context_test.cc

```diff
@@ -13,6 +13,17 @@
 #include "chainerx/native/native_device.h"
 #include "chainerx/testing/threading.h"
 
+#ifdef _WIN32
+int setenv(const char* name, const char* value, int overwrite) {
+    if (!overwrite) {
+        size_t required_count = 0;
+        auto err = getenv_s(&required_count, nullptr, 0, name);
+        if (err != 0 || required_count != 0) return err;
```

Comment:
Ditto.

Reply:
fixed it
https://github.com/durswd/chainer/commit/c522c857f7ec6a63d15dd2e91b0c6fe8e9f7b395